api/mesh/v1alpha1/rate_limit.proto

```diff
@@ -0,0 +1,68 @@
+syntax = "proto3";
+
+package kuma.mesh.v1alpha1;
+
+option go_package = "github.com/kumahq/kuma/api/mesh/v1alpha1";
+
+import "mesh/v1alpha1/selector.proto";
+
+import "google/protobuf/duration.proto";
+import "google/protobuf/wrappers.proto";
+import "validate/validate.proto";
+import "config.proto";
+
+message RateLimit {
+  // List of selectors to match dataplanes that rate limit will be applied for
+  repeated Selector sources = 1 [ (validate.rules).repeated .min_items = 1 ];
+
+  // List of selectors to match services that need to be rate limited.
+  repeated Selector destinations = 2
+      [ (validate.rules).repeated .min_items = 1 ];
+
+  message Conf {
+    message Http {
+      // The number of HTTP requests this RateLimiter allows
+      // +required
+      google.protobuf.UInt32Value requests = 1;
+
+      // The the interval for which `requests` will be accounted.
+      // +required
+      google.protobuf.Duration interval = 2;
```

Comment:
in validators, I think we need to check whether we set interval. User can omit this and it is require in Envoy config https://www.envoyproxy.io/docs/envoy/latest/api-v3/type/v3/token_bucket.proto so the config will be invalid